From de57c42da930470e62fb520f6c0de5d87da67385 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Thu, 29 Mar 2012 16:37:34 +0200
Subject: [PATCH] usb-ehci: Drop unused sofv value

The sofv value only ever gets a value assigned and is never used (read)
anywhere, so we can just drop it.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 hw/usb-ehci.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/hw/usb-ehci.c b/hw/usb-ehci.c
index 3934bf0..ff69587 100644
--- a/hw/usb-ehci.c
+++ b/hw/usb-ehci.c
@@ -403,7 +403,6 @@ struct EHCIState {
     /*
      *  Internal states, shadow registers, etc
      */
-    uint32_t sofv;
     QEMUTimer *frame_timer;
     int attach_poll_counter;
     int astate;                        // Current state in asynchronous schedule
@@ -1082,10 +1081,6 @@ static void ehci_mem_writel(void *ptr, target_phys_addr_t addr, uint32_t val)
         val &= USBINTR_MASK;
         break;
 
-    case FRINDEX:
-        s->sofv = val >> 3;
-        break;
-
     case CONFIGFLAG:
         val &= 0x1;
         if (val) {
@@ -2165,9 +2160,6 @@ static void ehci_frame_timer(void *opaque)
                 ehci_set_interrupt(ehci, USBSTS_FLR);
                 ehci->frindex = 0;
             }
-
-            ehci->sofv = (ehci->frindex - 1) >> 3;
-            ehci->sofv &= 0x000003ff;
         }
 
         if (frames - i > ehci->maxframes) {
